plugins/shrs_cd_stack/src/lib.rs

```diff
@@ -75,6 +75,7 @@ impl Plugin for CdStackPlugin {
         PluginMeta::new(
             "Cd Stack",
             "Provides the ability to quickly navigate directories like a stack.",
+            None
```

Comment:
Could you also open an issue for me to populate all of these help messages

Reply:
Done: https://github.com/MrPicklePinosaur/shrs/issues/411